.github/workflows/tests.yaml

```diff
@@ -1,15 +1,16 @@
 name: Unit Tests
 on:
+  workflow_dispatch:
   push:
     branches:
       - "**"
-    paths-ignore:
-      - "**/*.md"
+    paths:
```

Comment:
Only do tests if python files have changed.
What do you think about this? Should we run it always?